src/lib.rs

```diff
@@ -69,8 +69,13 @@ impl<BUS> CommonBus<BUS> {
     }
 
     fn lock<R, F: FnOnce(&mut BUS) -> R>(&self, f: F) -> R {
-        atomic::compare_exchange(&self.busy, false, true, Ordering::SeqCst, Ordering::SeqCst)
-            .expect("Bus conflict");
+        let compare =
+            atomic::compare_exchange(&self.busy, false, true, Ordering::SeqCst, Ordering::SeqCst)
+                .is_err();
```

Comment:
Hmm... I see that there definitely is implicit formatting going on, as the eventual panic will show the unwrapped error type. Given that it's obvious here, we definitely don't need it. Mind adding an entry into `CHANGELOG.md` (I just added this to `master`) tracking this change?